sdks/wasm/framework/src/WebAssembly.Net.Http/WasmHttpMessageHandler.cs

```diff
@@ -438,105 +418,4 @@ public override void Write (byte [] buffer, int offset, int count)
 			}
 		}
 	}
-
-	/// <summary>
-	/// Specifies a value for the 'credentials' option on outbound HTTP requests.
-	/// </summary>
-	public enum FetchCredentialsOption {
```

Comment:
Removing all this is in favor of the Blazor wrappers is nice for blazor but makes it harder when using the Actual bindings

Reply:
What do you recommend? Should we have a copy of the types \ extension methods in the Bindings project?